WcaOnRails/spec/mailers/competitions_mailer_spec.rb

```diff
@@ -301,4 +301,23 @@
       expect(mail.body.encoded).to include(link_to_competition_schedule_tab(competition))
     end
   end
+
+  describe "registration_reminder_email" do
+    let(:organizer) { FactoryBot.create(:user) }
+    let(:delegate) { FactoryBot.create(:delegate) }
+    let(:competitor) { FactoryBot.create(:user) }
+    let(:competition) { FactoryBot.create(:competition, name: "Comp of the future 2020", id: "CompFut2020", delegates: [delegate], organizers: [organizer]) }
```

Comment:
I think you can use here the `:with_delegate` and `:with organizer` traits instead of creating them by hand. Also, maybe something other than the mail directly should be added to indicate that the user has indeed bookmarked the competition?

Reply:
I was thinking about maybe adding a `bookmarked` trait to the competitions factory or something like that, but in hindsight I don't think it's worth it as it probably wouldn't be used anywhere else.